### PR TITLE
README.rst - Create title for config file

### PR DESCRIPTION
**Pull request by IanEdington**
I spent an unusually long time looking for this section of the docs.

The phrase `config file` is used throughout the docs but a section describing the config file is not present. This will make it easier for people to find the docs regarding config files.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1396,6 +1396,9 @@ fixed but without actually modifying them:
 
     $ php php-cs-fixer.phar fix /path/to/code --dry-run
 
+Config File
```
Comment (keradus) [inline README.rst]: file shall be lowercased

**Comment by keradus** [comment]
Hi !
This is a nice change indeed :+1:

Please, change target branch for 2.2.
Also, as we kindly ask you in contributing guidelines, don't manually modify the readme file. It's autogenerated from code and you shall change code itself, then autogenerate the readme file.
I expect CIs to fail due to that.

**Comment by Slamdunk** [comment]
@IanEdington the right place to apply the edit is here:

https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/e5c2abb7addf85bfc241d880d516c66c4203d7f2/src/Console/Command/HelpCommand.php#L116-L125

File: README.rst (hunk 1 above)
Comment (julienfalque) [reply README.rst]: Actually I'm not sure: https://en.wikipedia.org/wiki/Capitalization#Titles

Comment (keradus) [reply README.rst]: it's not a title, it's header.
even in link you provided, there are `By context`, `By name of style`, `Sentence case` and so

**Comment by IanEdington** [comment]
@keradus @Slamdunk @julienfalque Wow best experience on a commit I've had in ages!! Timely, polite, and with good feedback. I will make that change. 🥇

**Comment by keradus** [comment]
Sadly, you still need to regenerate the readme and commit it. We found no way to not have readme file included in repo and yet visible in github repo page.
please, run `php php-cs-fixer readme > README.rst` and commit the changes :+1:

**Comment by keradus** [comment]
Thank you @IanEdington.

File: README.rst (hunk 1 above)
Comment (julienfalque) [reply README.rst]: I thought this should be applied to headers as well, ok then :+1:

**Pull request closed without merging**